l3kernel/l3text-case.dtx

```diff
@@ -395,10 +426,19 @@
   {
     \@@_if_q_recursion_tail_stop_do:Nn #5
       { \@@_change_case_break: }
+    \str_if_eq:nnTF {#5} { \@@_change_case_exclude_word:n }
```

Comment:
Do we have a possibility to use a faster test than `\str_if_eq:nnTF`? Can the marker be a quark? If so an `\ifx`-based test would be preferable, if I skimmed over its usages correctly it should never actually be expanded, so its meaning can be arbitrary.

Reply:
I'd thought about this a bit - whilst I see your point, we are not in a place where performance at this level of detail is going to matter (I think) - and we know that quarks have a tendency to escape even though 'they will never be expanded' :)